### DISCUSSION TITLE
发送延迟消息时，不能发送超过三天的延迟消息

### DISCUSSION THREAD

**Discussion by Hybdream95** [apache/rocketmq#6831] [Q&A]
timer message illegal, the delay time should not be bigger than the max delay 259200000ms; or if set del msg, the delay time should be bigger than the current time
我取消了延迟等级后，发送延迟消息，发现超过三天的延迟消息创建不成功，原因是延迟时间不应大于最大延迟259200000ms，但是使用延迟等级就不会报这个问题。那么我应该怎么去解决这个问题？
broker的配置信息：
![1685345044564](https://github.com/apache/rocketmq/assets/127362348/d2e0478e-7ec1-4cbb-b89e-eaebb87e2c41)
brokerClusterName = DefaultCluster
brokerName = broker-a
brokerId = 0
deleteWhen = 04
fileReservedTime = 2160
brokerRole = ASYNC_MASTER
flushDiskType = ASYNC_FLUSH

proxy的配置信息：
![1685344762175](https://github.com/apache/rocketmq/assets/127362348/69a19728-0f7d-4315-8468-7965345d821c)
{
  "rocketMQClusterName": "DefaultCluster",
  "grpcServerPort":"8082",
  "namesrvAddr":"192.168.1.141:9876",
  "maxDelayTimeMills":"4,320,000,000",
  "useDelayLevel":false
}

**Comment by chenzlalvin** [comment]
Check if the configuration “ maxDelayTimeMills” has taken effect

**Reply by Hybdream95** [reply]
超过配置文件中 maxDelayTimeMills的时间会报
![1685427263567](https://github.com/apache/rocketmq/assets/127362348/443cd0ee-b475-49e2-b62a-6960e1170aaf)

**Comment by fuyou001** [comment]
通过mqadmin工具，修改broker的timerMaxDelaySec参数（注意持续化，要保确保重启不变），一般不建议设置这么长的延迟消息。

**Reply by Hybdream95** [reply]
解决了，设置了这个值后，没有报错了，谢谢大佬

**Reply by zhoumanchao** [reply]
大佬，在配置文件broker.conf里面加这个timerMaxDelaySec = 604800配置项吗？
